src/fvtest

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+# This script sets up the environment for an FV test, then runs the command
+# passed to it. For example `./fvtest ls -ltrh` will run `ls -ltrh` in the
+# fvtest environment.
+
+# Register a cleanup function that terminates memcached when the script exits.
+cleanup()
+{
+  if [[ $MEMCACHED_PORT != 0 ]]; then
+    kill -TERM $MEMCACHED_PID
+  fi
+}
+
+trap cleanup EXIT
+
+
+# Wait for a server to start accepting TCP connections.
+#
+# @param $1 - The time to wait in ms.
+# @param $2 - The port to wait for.
+#
+# @return   - 0 if the server started successfully, non-0 otherwise.
+wait_for_server()
+{
+  local timeout_ms=$1
+  local port=$2
+
+  local wait_time_ms=0
+  local sleep_time_ms=100
+
+  while [[ $wait_time_ms < $timeout_ms ]]; do
+    if nc -z "127.0.0.1" $port; then
+      break
+    fi
```

Comment:
A possibly better test would be to send a `version` command (which should return `VERSION` followed by a string) to check that the server is up and ready to receive requests.  This is what `libmemcached` does to check a server is active.